apps/web/scripts/kernel/handlers/solanaHandler.ts

```diff
@@ -78,3 +79,40 @@ export const signAndSendTransaction: HandleMethod = async ({
 		signatureString,
 	});
 };
+
+export const signTransactionAbstractionFee: HandleMethod = async ({
+	privateKey,
+	payload,
+	responseMethod,
+}) => {
+	const keypair = Keypair.fromSecretKey(privateKey);
+
+	const serializedTransaction = decode(payload.transaction as string);
+	const transaction = VersionedTransaction.deserialize(serializedTransaction);
+
+	transaction.sign([keypair]);
+
+	const txStr = base58.encode(transaction.serialize());
```

Comment:
`encode` already directly imported from `bs58` package, remember to remove `base58` import

Reply:
if i remove `base58`, it causes error